cmd/benchmark.go

```diff
@@ -330,6 +320,12 @@ func benchmarkFlags() *cli.Command {
 				Value: 100,
 				Usage: "number of small files",
 			},
+			&cli.IntFlag{
+				Name:    "threads",
+				Aliases: []string{"p"},
+				Value:   3,
```

Comment:
The default should be 1, to be backwards compatible.

Reply:
ebf2379